php-mode.el

```diff
@@ -1033,10 +1043,30 @@ PHP heredoc."
   "PHP Mode face used to highlight function names."
   :group 'php-faces)
 
+(defface php-function-caller-face '((t (:inherit default)))
+  "PHP Mode face used to highlight calling function names."
+  :group 'php-faces)
```

Comment:
@Fuco1 Thanks for your review.

I added `php-method-call` face. 
(`php-function-call` and `php-static-method-call` are unused, but I will use them later.)